### PR TITLE
[COST-3884] update_cost_model_costs for OCP only

### PR DESCRIPTION
**Pull request by djnakabaale**
## Jira Ticket

[COST-3884](https://issues.redhat.com/browse/COST-3884)

## Description
This change updates conditional logic to run only `update_cost_model_costs` for OCP.

## Testing

1. Checkout Branch
2. Restart Koku
3. Hit endpoint or launch shell
    1. You should see ...
4. Do more things...

## Notes

...

### CODE REVIEW

File: koku/masu/processor/tasks.py [hunk 1]
```diff
@@ -498,6 +498,10 @@ def update_summary_tables(  # noqa: C901
         Provider.PROVIDER_AWS_LOCAL,
         Provider.PROVIDER_AZURE,
         Provider.PROVIDER_AZURE_LOCAL,
+        Provider.PROVIDER_GCP,
+        Provider.PROVIDER_GCP_LOCAL,
+        Provider.PROVIDER_OCI,
+        Provider.PROVIDER_OCI_LOCAL,
     ):
```
Comment (cgoodfred) [inline koku/masu/processor/tasks.py:505]: Might be easier to just swap this to say If the provider isn't OCP. Would prevent us from having to update it in the future if we add any extra providers down the line

Comment (djnakabaale) [reply koku/masu/processor/tasks.py]: Thanks @cgoodfred, its cleaner this way.

File: koku/masu/test/processor/test_tasks.py [hunk 2]
```diff
@@ -674,10 +673,11 @@ def setUp(self):
         super().setUp()
         self.aws_accessor = AWSReportDBAccessor(schema=self.schema)
         self.ocp_accessor = OCPReportDBAccessor(schema=self.schema)
+        self.dh = DateHelper()
```
Comment (maskarb) [inline koku/masu/test/processor/test_tasks.py:676]: 100% nit-pick: The `MasuTestCase` inherits from `IamTestCase` which includes a `dh` attribute:
https://github.com/project-koku/koku/blob/main/koku/api/iam/test/iam_test_case.py#L71

So, you can remove this:
```suggestion
```

**Review by maskarb — APPROVED** [review]
LGTM!

**Review by maskarb — APPROVED** [review]

**Pull request merged**